### ISSUE TITLE
Install Question

### ISSUE THREAD

**Issue by theGuruWithin** [tiffany352/twitter-archive-browser#25]
Thank you for taking the time to create this browser... 

In Alpha 3 Assets there are 2 different .exe files.  Before I install to give it a test drive I was wondering the difference between the files and which one to install to use your Twitter-Archive-Browser....

Twitter-Archive-Browser-0.1.0-alpha3.exe 
and 
Twitter-Archive-Browser-Setup-0.1.0-alpha3.exe 

Thanks again

Much Gratitude,
A

**Comment by tiffany352** [comment]
The Setup will install it on your PC, while the other .exe is a portable version that you can run without installing. I'll try to make the naming more clear in future releases.

**Comment by theGuruWithin** [comment]
Thank you for your quick response!

Awesome... I was going to request a portable version - Perfect...

So do you just run .exe for portable version and that's IT? 

And can it be run from any folder?

Thanks again...

Total Respect and Total Gratitude

Love and Pranams,

A

**Comment by tiffany352** [comment]
Yeah, you can put it anywhere and run it. It takes a bit longer to start up than the installed version, and I'm not sure why that is.

**Comment by jayou360** [comment]
Is there a Mac version?

**Comment by tiffany352** [comment]
@jayou360 Unfortunately I can't create a mac build right now since I don't own a mac and it's the only OS that requires you build for macOS using macOS, unlike windows and linux.

Someone else also tried building it on their mac but it was having issues with crashing that I can't debug as well.

**Comment by jayou360** [comment]
Thanks for your reply. Is there any way to view the JSON's files using a browser script? I feel like I saw another version you did online but am having trouble finding it now.

**Comment by tiffany352** [comment]
@jayou360 There's an older version I made last year that still works, but has fairly barebones functionality. https://twitter.com/Tiffnixen/status/1175907702767144962

**Comment by jayou360** [comment]
That's perfect. Thanks!

**Comment by sepehr** [comment]
I just built it against node 12.13.0 on a Mojave.

- It throws an error when browsing an already extracted directory. 
- It crashes when I drag the zip file to the window.

![image](https://user-images.githubusercontent.com/23576/70644746-e5158200-1c43-11ea-8656-fe904629ec5f.png)

Here's the mac app in case someone needs it. 
https://gofile.io/?c=7bZuhQ
